### PR TITLE
BH-1233: Prepare readme.txt for wp.org

### PR DESCRIPTION
**Pull request by mindctrl**
## Description

This PR adds to the readme.txt so that it looks nice on wp.org.

- [x] Words and stuff
- [x] Screenshots

https://wpengine.atlassian.net/browse/BH-1233

### CODE REVIEW

File: readme.txt [hunk 1]
```diff
@@ -1,8 +1,45 @@
...
 Stable tag: 0.6.0
 License: GPLv2 or later
 License URI: https://www.gnu.org/licenses/gpl-2.0.html
 Author: WP Engine
+Contributors: antpb, apmatthe, chriswiegman, kevinwhoffman, markkelnar, matthewguywright, mindctrl, modernnerd, rfmeier, wpengine
+
+A WordPress plugin to create custom post types, custom fields, and custom taxonomies for headless WordPress sites.
+
+== Description ==
+Atlas Content Modeler (ACM) is a content modeling solution for WordPress. Using the intuitive interface, you can create custom post types, as well as custom fields and taxonomies for those post types, with ease.
```
Comment (narcher7) [inline readme.txt:14]: Replace "the intuitive" with "an intuitive."

File: readme.txt [hunk 2]
```diff
@@ -1,8 +1,45 @@
...
+### For Developers
+Developers get a modern content modeling system that automatically integrates with WPGraphQL and the WordPress REST API. No need to write code or install other plugins!
+
+### For Publishers
+Publishers get friendly and familiar content entry pages.
+
+== Installation ==
+This plugin can be installed directly from your WordPress site.
+
+1. Log in to your WordPress site and navigate to _Plugins &rarr; Add New.
```
Comment (narcher7) [inline readme.txt:25]: Bold **Plugins > Add new**

File: readme.txt [hunk 3]
```diff
@@ -1,8 +1,45 @@
...
+2. Type "Atlas Content Modeler" into the Search box.
+3. Locate the Atlas Content Modeler plugin in the list of search results and click **Install Now**.
+4. Once installed, click the Activate button.
+
+It can also be installed manually using a zip file.
+
+1. Download the Atlas Content Modeler plugin from WordPress.org.
+2. Log in to your WordPress site and navigate to _Plugins &rarr; Add New.
```
Comment (narcher7) [inline readme.txt:33]: Same note as line 25

File: readme.txt [hunk 4]
```diff
@@ -1,8 +1,45 @@
...
+3. Click the "Upload Plugin" button.
```
Comment (narcher7) [inline readme.txt:34]: Make "Upload Plugin" bold. Remove quotations.

File: readme.txt [hunk 5]
```diff
@@ -1,8 +1,45 @@
...
+4. Click the "Choose File" button, select the zip file you downloaded in step 1, then click the "Install Now" button.
```
Comment (narcher7) [inline readme.txt:35]: Same note as 34 for **Choose File**.

File: readme.txt [hunk 6]
```diff
@@ -1,8 +1,45 @@
...
+5. Click the "Activate Plugin" button.
```
Comment (narcher7) [inline readme.txt:36]: Same note as 34 for **Activate Plugin**.

File: readme.txt [hunk 7]
```diff
@@ -1,8 +1,45 @@
...
+
+== Screenshots ==
+1. (work in progress) Description here... screenshot-1.png in assets/wporg
+2. screenshot-2.png in assets/wporg
+3. screenshot-2.png in assets/wporg
+
+== Frequently Asked Questions ==
+= Can Atlas Content Modeler be used with traditional WordPress sites? =
+ACM is primarily intended for headless WordPress applications. For that reason, the WordPress REST API and WPGraphQL are the only two officially supported APIs. That said, it is possible to fetch the data for your models in a traditional, monolithic WordPress site by using the `rest_do_request()` PHP function that the REST API provides, or the `graphql()` PHP function that WPGraphQL provides.
```
Comment (narcher7) [inline readme.txt:45]: Remove the term "monolithic". Hashim has requested we avoid the term.

Comment (narcher7) [reply readme.txt:45]: Remove comma after "... that the REST API provides".

**Review by narcher7 — APPROVED** [review]

**Comment by mindctrl** [comment]
@narcher7 thanks for the review. I've updated per your recommendations.

File: readme.txt [hunk 8]
```diff
@@ -1,8 +1,45 @@
...
+3. Click the **Upload Plugin** button.
+4. Click the **Choose File** button, select the zip file you downloaded in step 1, then click the **Install Now** button.
+5. Click the **Activate Plugin** button.
+
+== Screenshots ==
+1. (work in progress) Description here... screenshot-1.png in assets/wporg
+2. screenshot-2.png in assets/wporg
+3. screenshot-2.png in assets/wporg
+
+== Frequently Asked Questions ==
```
Comment (ChrisWiegman) [inline readme.txt:43]: Do we want a second question that links back to GitHub for support/etc?

Comment (mindctrl) [reply readme.txt:43]: @ChrisWiegman good idea. Added in https://github.com/wpengine/atlas-content-modeler/pull/289/commits/0dc194a804361ca147e5f85f77c8368f6a4e8de2

**Review by nickcernis — COMMENTED** [review]
Thanks for your work on this, John! It reads well to me. Happy to approve with the screenshots.

**Review by ChrisWiegman — APPROVED** [review]
Thanks for the GH addition. I realize most will still open support tickets on .org, and that's OK. Pointing them back to the repo, for those who want it, might really help though.

The rest of this looks damn good to me. Thanks for your work on this!

**Comment by mindctrl** [comment]
@ChrisWiegman @nickcernis screenshots and descriptions have been added.

**Review by nickcernis — APPROVED** [review]
@mindctrl Looks good! Nice choice of screenshots.

**Review by ChrisWiegman — APPROVED** [review]

**Pull request merged**